### DISCUSSION TITLE
[Q&A] 为什么 chat-next-web 和 lobehub 对于模型信息的提问返回不一样？

### DISCUSSION THREAD

**Discussion by Nash-x9** [lobehub/lobehub#727] [General Question] [answered]
### 🧐 问题描述 | Proposed Solution

采用azure api，分别部署了chat-next-web和lobehub，询问他们的模型版本，返回不一致，特别是使用gpt4时，lobehub总是返回他是gpt3模型。

### 📝 补充信息 | Additional Information

下图是，分别使用gpt3-turbo、gpt4、gpt4-32k询问结果。
<img width="1482" alt="image" src="https://github.com/lobehub/lobe-chat/assets/12541919/0e7690ac-4044-4ad8-a2d2-d290dceb710e">
<img width="1139" alt="image" src="https://github.com/lobehub/lobe-chat/assets/12541919/c632eaff-d35e-4aad-9eeb-e4931d17171b">
<img width="1110" alt="image" src="https://github.com/lobehub/lobe-chat/assets/12541919/093961cc-d6f2-4ce7-a395-3d1c14f0f182">

[a comment by an automated account is not shown]

**Comment by canisminor1990** [comment, marked as answer]
在 chat-next-web 中将模型信息作为了 预设 prompt 传递给了 gpt, 在 lobechat 中为了保持 systemrole 纯净并没有做这个预处理

[a comment by an automated account is not shown]
